app/views/tracks/community_solutions/show.html.haml

```diff
@@ -19,7 +19,7 @@
         .seperator /
         = link_to "Community Solutions", track_exercise_solutions_path(@track, @exercise)
         .seperator /
-        .exercise-title #{@author.handle}'s solution
+        .exercise-title #{@own_solution ? 'Your' : "#{@author.handle}'s"} solution
```

Comment:
It has always bothered me that my own community solution was visually not different from another user's community solution, except for some extra options. What do you think of this change?

Reply:
Yep!